File: python/hamming/hamming.py
def distance(strand_a, strand_b):
	if len(strand_a) != len(strand_b):
		raise ValueError("strands are different lengths")

	hamming = [x for x in range(0, len(strand_a)) if strand_a[x] != strand_b[x]]

	return len(hamming)

File: csharp/hamming/Hamming.cs
using System;

public static class Hamming
{
    public static int Distance(string firstStrand, string secondStrand)
    {
    	if (firstStrand.Length != secondStrand.Length){
    		throw new ArgumentException("Incorrect string lengths");	
    	}

    	if(firstStrand == secondStrand){
    		return 0;
    	}

    	int hammingDistance = 0;

    	for(int i = 0; i < firstStrand.Length; i++){
    		if(firstStrand[i] != secondStrand[i]){
    			hammingDistance++;
    		}
    	}
    	
    	return hammingDistance;
    }
}